### DISCUSSION TITLE
20240223-intern-review/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#50] [Comments]
# 20240223-intern-review/

6주동안 인프랩 데브옵스 팀으로써 일하면서 배운 것들 또는 경험한 것들을 공유합니다.

https://tech.inflab.com/20240223-intern-review/

**Comment by Yoochanhong** [comment]
정말 대단합니다..!

**Comment by ghkdqhrbals** [comment]
문제에 대한 정의부터 시작하여 다양한 해결방법들을 팀원들과 함께 진행한 경험은 정말 값지다고 생각합니다. 앞으로 멋진 개발자로 성장하시길 응원합니다.
